param_templates/MOM_input.yaml

```diff
@@ -3633,28 +3354,28 @@ Global:
             INPUT         - Testing with fixed values.
         datatype: string
         value:
-            $COMP_WAV in ["ww3", "ww3dev"] and $WAV_GRID == "wtx0.66v1" : "COUPLER"
+            $COMP_WAV in ["ww3", "ww3dev"] and $WAV_GRID == "wtx2_3v2" : "COUPLER"
     STOKES_DDT:
         description: |
             default = False
             Flag to use Stokes d/dt
         datatype: string
         value:
-            $COMP_WAV in ["ww3", "ww3dev"] and $WAV_GRID == "wtx0.66v1" : True
+            $COMP_WAV in ["ww3", "ww3dev"] and $WAV_GRID == "wtx2_3v2" : True
```

Comment:
Change to $WAV_GRID in ["wtx2_3v2", "wgx3v7"] here and lines 3371, 3378, 3387, and 3394.